### PR TITLE
Combat Refactor

### PR DESCRIPTION
**Pull request by Theonewhowaits**
## About The Pull Request

Greatly alters weapon damage.
Many weapons have their damage halved or just heftily reduced. To try and give ballistics a reason to be used, increased many of their magazines. Note, energy weapons have recharging cells and tend to be smaller in size. Mob health has been similar halved. Well combat mobs, I am not going through 100 files of rp mobs and things I unsure can be spawned. Also not messing with mobs apart of PRs, rather not be that guy.

## Changelog

:cl:
balance: rebalances combat mobs health alongside weapon damage
/:cl:

### CODE REVIEW

**Comment by Aroliacue** [comment]
Hey, this seems pretty cool. Are you on our Discord server? I'd like to discuss this further

**Comment by Theonewhowaits** [comment]
Yes but I wish for my discord to remain private. I will attempt to watch careful here and in those chats. Then upon processing feedback and discussion, make further changes.

File: code/modules/mob/living/simple_mob/subtypes/mechanical/hivebot/hivebot.dm [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Hivebots are tuned towards how many default lasers are needed to kill them.
 // As such, if laser damage is ever changed, you should change this define.
-#define LASERS_TO_KILL * 40
+//chompedit: Reduce health of hivebots by reducing 40 to 15
+#define LASERS_TO_KILL * 15
```
Comment (Raeschen) [inline code/modules/mob/living/simple_mob/subtypes/mechanical/hivebot/hivebot.dm:4]: Put the chompEDIT comment after, not above, for single line edits.

Comment (TheGreatKitsune) [reply code/modules/mob/living/simple_mob/subtypes/mechanical/hivebot/hivebot.dm:4]: That is to say on the same line I believe.

**Comment by Razgriz1032** [comment]
Seems like the a57 ammo is not being touched for the P90. Is that intentional?

**Comment by Theonewhowaits** [comment]
Unintended.
I will remedy this after I am able to do a secondary reading of the material.

**Comment by Darlantanis** [comment]
This is an extensive enough rebalancing that I cannot predict the results. I think it would be a good idea to test merge and gather feedback from our more combat-interested players (and ideally a couple event managers).

Hopefully this will be a step in the right direction. Our combat system direly needs improvement, but a full rework seems unlikely in the near future.

**Comment by Kazzc** [comment]
This looks very promising, curious how this will affect events.

**Comment by Theonewhowaits** [comment]
I see I have gathered the gaze of the triumvirate.
The test merge has been approved, and activated.
Things are ahead of schedule, but I am prepared to dance.
Let us hope all goes well, and what slipped by me is remedy quickly.

**Comment by tacoguy7765093** [comment]
Pretty minor but the 7.62 stripper clips are much bigger then the bolt action rifle it's meant for and the sprite does not reflect properly, still looking full when half empty. If intentional, I can possibly change the code behind the icon.

**Comment by Darlantanis** [comment]
Two things from the staff dev channel:

1. Tunneler spider override doesn't work. Spider type is "giant_spider/tunneler,' override here is "giant_spider/tunneler_spider." This creates a new spider type instead of overriding the tunneler health values.

2. It's been raised that mechs haven't been adjusted to reflect these changes, meaning mechs will be more potent against mobs while also being much harder to destroy. More of a heads-up, no suggestions yet.

**Comment by Raeschen** [comment]
I had to merge master into this branch to make the TM happy. Pull your master.

**Comment by Raeschen** [comment]
This is looking pretty good, the feedback is positive.

I think that the mech stuff is handled by virtue of the projectiles already being edited? Mechs were the only remaining remark.

We're gunna keep this merged for an event tonight to see what the EM thinks of it but it's looking like a merge soon enough. Nice work.

**Pull request merged**